packages/crypto/src/__tests__/crypto-test.ts

```diff
@@ -0,0 +1,21 @@
+import { test, expect, describe } from "@jest/globals";
+import { uint8ArrayFromHexString } from "@turnkey/encoding";
+import { getPublicKey, generateP256KeyPair } from "../crypto";
+
+describe("Crypto Utils - Exported Functions", () => {
```

Comment:
would be nice to test other functions that we export as well (a large e2e-like test could do the trick)